packages/trackerless-network/src/logic/ContentDeliveryManager.ts

```diff
@@ -22,8 +22,8 @@ import { ProxyDirection, StreamMessage, StreamPartitionInfo } from '../proto/pac
 import { EntryPointDiscovery, NETWORK_SPLIT_AVOIDANCE_LIMIT } from './EntryPointDiscovery'
 import { Layer0Node } from './Layer0Node'
 import { Layer1Node } from './Layer1Node'
-import { RandomGraphNode } from './RandomGraphNode'
-import { createRandomGraphNode } from './createRandomGraphNode'
+import { ContentDeliveryLayerNode } from './ContentDeliveryLayerNode'
+import { createContentDeliveryLayerNode } from './createContentDeliveryLayerNode'
 import { ProxyClient } from './proxy/ProxyClient'
 
 export type StreamPartDelivery = {
```

Comment:
Probably need to think about the name of the type `StreamPartDelivery` at some point.